OpenRA.Mods.Common/Pathfinder/PathGraph.cs

```diff
@@ -84,6 +84,7 @@ sealed class PathGraph : IGraph<CellInfo>
 
 		readonly CellConditions checkConditions;
 		readonly LocomotorInfo locomotorInfo;
```

Comment:
I doubt caching `locomotorInfo` separately is worth it now. Can you remove this and use `locomotor.Info` directly instead?